test/cljs/status_im/test/utils/utils.cljs

```diff
@@ -0,0 +1,14 @@
+(ns status-im.test.utils.utils
+    (:require [cljs.test :refer-macros [deftest is]]
+              [status-im.utils.utils :as u]))
+
+(def count (atom 0))
```

Comment:
I think it would be better to initialise/tear-down stateful test data in fixtures, that way, it's less error prone when tests are run multiple times/just some of the test functions are run repeatedly.
Maybe non issue for clojurescript tests which are always run whole through test runner, but still think it's nice to keep the pattern (maybe for future move to .cljc, leveraging jvm clojure for unit-tests, etc)